src/components/molecules/DeleteWarningPopover.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 
-import { PanelVariant, PopoverPanel } from '@altinn/altinn-design-system';
-import { Button, ButtonColor, ButtonVariant } from '@digdir/design-system-react';
+import { Button, Popover } from '@digdir/design-system-react';
```

Comment:
This change seems to fix the problem in Percy that made me add a workaround for `data-radix-popper-content-wrapper` in percy.css. Could you remove that workaround in the process? 🙏